### PR TITLE
Rescue Module update + other changes

### PR DESCRIPTION
**Pull request by Chada-1**
Been awhile since I made a PR actually.

I gave the Rescue module a jetpack and a rescue version of the mobility module, the speedboost is the same speed as the VTEC that they'd normally have to be upgraded to get. The Jetpack has no way to refuel but has enough that they'd have to need to use it a LOT (Hopefully 30+ minutes) to run out of fuel. Not very good design but there's no easy way that I see to make it refuel (I'm clueless).

Also fixed multiple other things, like other 'borgs/drones being given dysfunctional jetpacks and the mining drone not having a drill on roundstart, which made them unable to mine outside of industrial drills. Mining and Rescue are now ineligible for the jetpack upgrade due to already having one.

And finally -- I gave the Clerical roleplay tools to afew of the other 'borgs too, it never made sense to isolate them to the Clerical only, their only usecase is to enhance roleplay, we shouldn't be isolating roleplay tools to a single module like that I think, so I gave Mining, Research and Service the same core tools. I would've removed Clerical, but players want me to keep it according to one of my feedback threads.

TLDR: _Reworked Rescue with Jetpack and speedboost, fixed Mining drone and jetpacks, gave clerical tools to Mining/Service/Research._

### CODE REVIEW

**Comment by Chada-1** [comment]
!review

**Comment by Chada-1** [comment]
Also this kinda sorta counts as a feedback thread? [Click here](https://forums.aurorastation.org/topic/15077-rescue-borg-fixes/?tab=comments#comment-142270)

**Review by Arrow768 — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]
